modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java

```diff
@@ -199,6 +199,39 @@ public void testGetLocalizationXml() {
 			xml.contains("default-locale=\"" + LocaleUtil.getDefault() + "\""));
 	}
 
+	@Test
+	public void testGetLocalizationXmlFromPreferences() throws Exception {
+		PortletPreferences preferences = new PortletPreferencesImpl();
+		MockPortletRequest portletRequest = new MockPortletRequest();
+
+		LocalizationUtil.setPreferencesValue(
+			preferences, "test", _ENGLISH_LANGUAGE_ID, "changedValue");
+
+		String xml = LocalizationUtil.getLocalizationXmlFromPreferences(
+			preferences, portletRequest, "test", "testValue");
+
+		Assert.assertTrue(
+			"PortletPreferences not properly applied to XML",
+			xml.contains(
+				"<test language-id=\"" + _ENGLISH_LANGUAGE_ID +
+					"\">changedValue</test>"));
```

Comment:
@kyle-miho According to 8h of https://docs.google.com/document/d/1UdrIWd4_anL3p0TwOttBwn0kIPWLNxk0cJG22qPfCgo/edit#heading=h.s6tdya70tg54, calling getLocalizationXmlFromPreferences here is supposed to return "testValue", not "changedValue". I'm pretty sure this is a typo in the test plan, but can you confirm?

Reply:
Yea thats a typo in the testplan, the way in the integration test should be the correct interaction, i updated the test plan